chef/cookbooks/suse-manager-client/recipes/default.rb

```diff
@@ -36,11 +38,9 @@
   EOH
 end
 
-# XXX requires chef-client with CHEF-4090 fixed otherwise the package
-# provider can't handle the URL
-package "https://#{manager_server}/pub/bootstrap/sm-client-tools.rpm"
-
-execute "sm-client" do
-  command "sm-client --hostname #{manager_server} --activation-keys #{activation_key}"
+# FIXME we should do it just the first time, otherwise system is registered again
+bootstap_script = "bootstrap-sles12."
+bootstap_script += node[:platform_version].split(".")[1]
```

Comment:
I would interpolate here as the size is still OK for one line. and use `last` instead of `[1]`

`"bootstrap-sles12.#{node[:platform_version].split(".").last}"`